test/ui-test/src/screens/StatusChatScreen.py

```diff
@@ -196,8 +196,8 @@ def reply_to_message_at_index(self, index: int, message: str):
         self.send_message(message)
     
     def edit_message_at_index(self, index: int, message: str):
-        message_object_to_edit = get_obj(ChatComponents.CHAT_LOG.value).itemAtIndex(int(index))
-        hover_obj(message_object_to_edit)
+        message_object_to_edit = wait_and_get_obj(ChatComponents.CHAT_LOG.value).itemAtIndex(int(index))
+        move_mouse_over_object(message_object_to_edit)
```

Comment:
@noeliaSD I needed to make these two changes to make this test pass reliably when run standalone.

Reply:
Realized `get_message_at_index` does exactly that.